data_safe_haven/resources/workspace/workspace.cloud_init.mustache.yaml

```diff
@@ -1,6 +1,11 @@
 #cloud-config
 
 write_files:
+  - path: "/etc/clamav/freshclam-mirror.conf"
+    permissions: "0400"
+    content: |
+      PrivateMirror {{ clamav_mirror_hostname }}
```

Comment:
Would be good to do this in Ansible. However, as there are other instances in cloud init let's open a new issue for that.

Reply:
We need a way to get these variables into the desired state file. This could be uploading a file with variables into blob storage or it could be writing them to the cloud-init. As you say, this is done in a few other places so that's the pattern I copied.

Reply:
Captured in #2075